src/main/java/seedu/staffsnap/logic/parser/ApplicantBookParser.java

```diff
@@ -52,6 +57,9 @@ public Command parseCommand(String userInput) throws ParseException {
         // Lower level log messages are used sparingly to minimize noise in the code.
         logger.fine("Command word: " + commandWord + "; Arguments: " + arguments);
 
+        IsConfirmed = IsConfirmedNext;
+        IsConfirmedNext = false;
```

Comment:
Can consider abstracting this away into a `setIsConfirmed()` method that updates the static field `isConfirmed` and resets `isConfirmedNext` to false.

Doing so maintains SLAP within ApplicantBookParser.java.